packages/lib-ts/as-pect.config.js

```diff
@@ -22,9 +22,28 @@ export default {
     const myImports = {
       env: {
         memory,
-        'console.log': (ptr) => {
-          const string = exports.__getString(ptr)
-          console.log(string)
+      },
+      log: {
+        _log: (level, msgPtr) => {
+          const msg = exports.__getString(msgPtr)
+          switch (level) {
+            case 0:
+              throw new Error(`[CRITICAL] ${msg}`)
```

Comment:
Is this how it works in the graph? To me is counterintuitive

Reply:
Yeap: https://thegraph.com/docs/en/subgraphs/developing/creating/graph-ts/api/#logging-api

> `log.critical(fmt: string, args: Array<string>): void` – logs a critical message and terminates the Subgraph.